po/fr.po

```diff
@@ -120,95 +121,90 @@ msgstr "ce texte d'aide"
 
 #: newsboat.cpp:103
 msgid "remove unreferenced items from cache"
-msgstr ""
+msgstr "retirer du cache les éléments non référencés"
 
 #: newsboat.cpp:128
-#, fuzzy
 msgid "Files:"
-msgstr "Fichier : "
+msgstr "Fichiers : "
 
 #. / This is printed out by --help before the path to the config file
 #: newsboat.cpp:130
-#, fuzzy
 msgid "configuration"
-msgstr "Chargement de la configuration..."
+msgstr "configuration"
 
 #. / This is printed out by --help before the path to the urls file
 #: newsboat.cpp:132
-#, fuzzy
 msgid "feed URLs"
-msgstr "Liens"
+msgstr "URLS de fils"
 
 #. / This is printed out by --help before the path to the cache file
 #: newsboat.cpp:134
-#, fuzzy
 msgid "cache"
-msgstr "<fichier_cache>"
+msgstr ""
```

Comment:
Even if the translation looks exactly like the English original, it's better to spell it out — this will make the translated/fuzzy/untranslated statistic more meaningful, and you won't have to trip up over this untranslated message in Poedit ;)